conf/application.conf

```diff
@@ -296,3 +296,14 @@ alert {
         lockDuration = 5 seconds
     }
 }
+
+
+renotify {
+    sdes {
+        enabled = true
+        cron = "0 0 7 ? * *"  # every day at 7am
+        destinations = ["Dms", "DataStore"]
```

Comment:
This (and `alert.sdes.destination` also)  should now list new `HmrcIlluminate` destination

Reply:
should we update the ticket?

Reply:
I think the idea was to implement ticket as a patch whereas this aims for `main`, so no need to update ticket imho.